infra/terraform/environments/alpha/main.tf

```diff
@@ -86,6 +86,8 @@ module "container_registry" {
 
 module "data_access" {
     source = "../../modules/data_access"
+
+    region = "${var.region}"
     env = "${var.env}"
     sns_arn = "${var.sns_arn}"
```

Comment:
So, it was added in another PR but it's used by the [lambda function](https://github.com/ministryofjustice/analytics-platform-ops/blob/master/infra/terraform/modules/data_access/lambda.tf#L22) to build the [SNS topic where GH events are published to](https://github.com/ministryofjustice/analytics-platform-ops/blob/master/infra/terraform/modules/data_access/github_webhooks_handler/github_webhooks.py#L49).